.travis.yml

```diff
@@ -24,3 +24,5 @@ after_success:
   - bin/createcoverage
   - pip install coverage==3.7.1 coveralls
   - coveralls
+  - docker.io build --tag plone.restapi:$(git rev-parse HEAD) .
+#  - docker.io push kitconcept/plone.restapi
```

Comment:
Ok, but if you want travis to push you need docker installed, right ?

Yeah, this is when you install docker from the ubuntu repositories, but if you for example install docker from the docker repos it is called docker.

I do not know how travis uses/installs docker there are different ways.